tests/functional/tests.d/347-proxyjump.sh

```diff
@@ -0,0 +1,467 @@
+    # Delete groupAddServer entries with missing proxy-port
+    plgfail groupDelServer_without_proxy_port $a1 --osh groupDelServer --group $group1 --host 192.168.3.100 --user testuser --port 22 --proxy-host 10.0.0.3 --proxy-user testuser
+    json .command groupDelServer .error_code ERR_MISSING_PARAMETER
+
+    # Delete groupAddServer entries with missing proxy-host
+    plgfail groupDelServer_without_proxy_host $a1 --osh groupDelServer --group $group1 --host 192.168.3.100 --user testuser --port 22 --proxy-port 22
+    json .command groupDelServer .error_code ERR_MISSING_PARAMETER
+
+    # Delete groupAddServer entries with missing proxy-user
+    plgfail groupDelServer_without_proxy_user $a1 --osh groupDelServer --group $group1 --host 192.168.3.100 --user testuser --port 22 --proxy-host 10.0.0.3 --proxy-port 22
+    json .command groupDelServer .error_code ERR_MISSING_PARAMETER
+    contain "When --proxy-host is specified, --proxy-user becomes mandatory"
+
+    # Delete groupAddServer entries
+    success groupDelServer_with_proxy $a1 --osh groupDelServer --group $group1 --host 192.168.3.100 --user testuser --port 22 --proxy-host 10.0.0.3 --proxy-port 22 --proxy-user testuser
+    json .command groupDelServer .error_code OK
+
+    success groupDelServer_with_proxy_hostname $a1 --osh groupDelServer --group $group1 --host 192.168.3.101 --user testuser --port 22 --proxy-host localhost --proxy-port 4444 --proxy-user testuser
+    json .command groupDelServer .error_code OK
+
+    #
+    # Test that proxy information is displayed in access lists
+    #
+
+    # Add self access with proxy
+    success add_access_for_list_check $a0 --osh selfAddPersonalAccess --host 192.168.1.200 --user listtest --port 2222 --proxy-host 10.0.0.5 --proxy-port 5555 --proxy-user listtest --force
+    json .command selfAddPersonalAccess .error_code OK
+
+    # Check that selfListAccesses shows the proxy information
+    success selfListAccesses_shows_proxy $a0 --osh selfListAccesses
+    json .command selfListAccesses .error_code OK
+    contain '"ip":"192.168.1.200"'
+    contain '"port":"2222"'
+    contain '"user":"listtest"'
+    contain '"proxyIp":"10.0.0.5"'
+    contain '"proxyPort":"5555"'
+
+    # Clean up
+    success cleanup_list_test $a0 --osh selfDelPersonalAccess --host 192.168.1.200 --user listtest --port 2222 --proxy-host 10.0.0.5 --proxy-port 5555 --proxy-user listtest
+    json .command selfDelPersonalAccess .error_code OK    # Add self access with proxy and proxy-user
+    success add_access_with_proxy_user_for_list_check $a0 --osh selfAddPersonalAccess --host 192.168.1.201 --user listtest --port 2222 --proxy-host 10.0.0.5 --proxy-port 5555 --proxy-user proxyuser --force
+    json .command selfAddPersonalAccess .error_code OK
+
+    # Check that selfListAccesses shows the proxy-user information
+    success selfListAccesses_shows_proxy_user $a0 --osh selfListAccesses
+    json .command selfListAccesses .error_code OK
+    contain '"ip":"192.168.1.201"'
```

Comment:
```suggestion
    json .value[0].ip 192.168.1.201
```
You can replace this line and all the other similar ones by using `json` parsing, way more clean/robust than `contain` (which is just a `grep` and is more used for things such as grepping for text messages in parts of the output we don't control directly or which is not reflected in the json return, etc.)